specs/paymentrequest.html

```diff
@@ -425,6 +425,13 @@
             length of 1, then set <a><code>shippingOption</code></a> to the <code>id</code> of
             the only <a><code>ShippingOption</code></a> in the sequence.
           </li>
+          <li>
+            If <code>details</code> contains a <code>shippingOptions</code> sequence with a
+            length greater than 1, and if any <a><code>ShippingOption</code></a> in the sequence
+            has the <code>selected</code> field set to <code>true</code>, then set
+            <a><code>shippingOption</code></a> to the <code>id</code> of the last <a><code>ShippingOption</code></a>
+            in the sequence with <code>selected</code> set to <code>true</code>.
+          </li>
```

Comment:
Sure, that's fine. I just want such behavior to be technically conformant. The conformance section of this document denotes everything except a named list of items as normative, and this step -- which specifies that the shippingOption is set a certain way -- doesn't fall into any of the named non-normative categories. That makes performing such action mandatory for conformance purposes, and I think that's too strict.